client/js/shout.js

```diff
@@ -197,6 +197,23 @@ $(function() {
 				data.msg
 			]);
 
+		chan.find(".messages .text:not(.processed)")
+			.each(function() {
+				var html = $(this).html();
+				$(this)
+					.addClass("processed")
+					.html(html.replace(/(\W)(#[\w\d-.]{0,200})/ig,
```

Comment:
> - I couldn't find a network where &channel is working. Are you sure that this is supported by shout's backend?

Typing `/join <whatever>` in the input [is simply a forward to Slate with `<whatever>` as an argument](https://github.com/erming/shout/blob/98344b2cac540eb5d862e4bf1c53e73a538c9afd/src/plugins/inputs/join.js), which in turn [merely sends `JOIN <whatever>`](https://github.com/slate/slate-irc/blob/6f2c820ee6afd28b935deafae99c1760b14e6341/index.js#L239) to the network, so I don't think there any massaging on that. Actually, it might be nice to prevent trying to join a network that does not respect the RFC, either on Shout or on Slate, but that's for another PR ;-)

Additionally, I asked @deiu as I know he is using Shout on a `&`-enabled server and he confirmed it works fine.

> - Actually, I do think so. Test it on the shout demo network for example. `/join #` works just fine.

You are correct (and it highlighted #483 ^^), so the right interval is `{0,199}` indeed!